static/oas/v1/stats/venues.yml

```diff
@@ -0,0 +1,101 @@
+paths:
+  venues:
```

Comment:
If I am following past Pull Requests correctly, this will need to be /venues.

Reply:
It can be either one, but `/venues` needs to be used if there's endpoints for both `/venues` (for Get All and Post) and `/venues:id` (for Get By ID and PUT)

However since this endpoint only has a Get All and Post, either one works

Reply:
I got the syntax wrong on this, Get By ID and PUT should be under `/venues/{venueId}`